Modules/Numerics/Optimizersv4/include/itkConjugateGradientLineSearchOptimizerv4.h

```diff
@@ -78,6 +78,41 @@ class ITK_TEMPLATE_EXPORT ConjugateGradientLineSearchOptimizerv4Template
   void
   StartOptimization(bool doOnlyInitialization = false) override;
 
+  /**
+   * Set/Get the learning rate for optimization. It is overridden by
+   * automatic learning rate estimation if enabled via a ScalesEstimator.
+   * The learning rate is updated during optimization according to the golden
+   * section line search, even if no ScalesEstimator is set.
+   *
+   * The learning rate set by this method is stored and re-used in subsequent
+   * calls to StartOptimization() unless overridden by automatic learning rate
+   * estimation.
+   */
+  void
+  SetLearningRate(TInternalComputationValueType learningRate) override;
+
+  /**
+   * Set/Get the learning rate for optimization. It is overridden by
+   * automatic learning rate estimation if enabled via a ScalesEstimator.
+   * The learning rate is updated during optimization according to the golden
+   * section line search, even if no ScalesEstimator is set.
+   *
+   * The learning rate set by this method is stored and re-used in subsequent
+   * calls to StartOptimization() unless overridden by automatic learning rate
+   * estimation.
+   */
+  const TInternalComputationValueType &
+  GetLearningRate() const override;
```

Comment:
Why do we need a getter method if it is defined in the base class? https://github.com/InsightSoftwareConsortium/ITK/blob/25d316237131813a37ff9c6eb4c634e6b55b2202/Modules/Numerics/Optimizersv4/include/itkGradientDescentOptimizerv4.h#L109-L114

Reply:
The method is not really needed but I didn't know another way to make the Set / Get docstrings consistent as requested. Is there a better way to accomplish this?

Reply:
Please, remove the method then. The comment about the Set/Get docstring only applies to cases where both methods are present.

Reply:
It's removed